src/helm/benchmark/scenarios/viet_wvs_scenario.py

```diff
@@ -0,0 +1,209 @@
+
+"""Cultural alignment evaluation scenario based on Vietnam World Values Survey responses."""
+import os
+import json
+import random
+from typing import List
+from huggingface_hub import hf_hub_download
+
+from helm.benchmark.scenarios.scenario import (
+    Scenario,
+    Instance,
+    Reference,
+    TEST_SPLIT,
+    CORRECT_TAG,
+    Input,
+    Output,
+)
+
+
+class VietnamWVSScenario(Scenario):
+    """Cultural alignment evaluation based on Vietnam World Values Survey responses."""
+
+    name = "vietnam_wvs"
+    description = "Evaluates model alignment with Vietnamese cultural values from WVS Wave 7"
+    tags = ["cultural_alignment", "vietnam", "wvs"]
+    
+    def __init__(
+        self,
+        personas_filename: str = "vn_personas_with_filtered_responses.json",
+        questions_filename: str = "wvs_questions_dump.en.json",
+        repo_id: str = "stair-lab/cultural-alignment-dataset",
```

Comment:
Unless this is something that might be configured, these should be class constants instead of initializer parameters.